plugins/gcp-projects/src/components/ProjectDetailsPage/ProjectDetailsPage.tsx

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Button,
+  ButtonGroup,
+  LinearProgress,
+  makeStyles,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableRow,
+  Theme,
+  Typography,
+} from '@material-ui/core';
+import {
+  useApi,
+  googleAuthApiRef,
+  HeaderLabel,
+  Page,
+  Header,
+  pageTheme,
+  SupportButton,
+  Content,
+  ContentHeader,
+} from '@backstage/core';
+import React from 'react';
+import { useAsync } from 'react-use';
+import { GCPApiRef } from '../../api';
+
+const useStyles = makeStyles<Theme>(theme => ({
+  root: {
+    maxWidth: 720,
+    margin: theme.spacing(2),
+  },
+  title: {
+    padding: theme.spacing(1, 0, 2, 0),
+  },
+  table: {
+    padding: theme.spacing(1),
+  },
+}));
+
+const DetailsPage = () => {
+  const api = useApi(GCPApiRef);
+  const googleApi = useApi(googleAuthApiRef);
+  const token = googleApi.getAccessToken(
+    'https://www.googleapis.com/auth/cloud-platform.read-only',
+  );
+
+  const classes = useStyles();
+  const status = useAsync(
+    () =>
+      api.getProject(
+        decodeURIComponent(location.search.split('projectId=')[1]),
+        token,
+      ),
+    [location.search],
+  );
+
+  if (status.loading) {
+    return <LinearProgress />;
+  } else if (status.error) {
+    return (
+      <Typography variant="h6" color="error">
+        Failed to load build, {status.error.message}
+      </Typography>
+    );
+  }
+
+  const details = status.value;
+
+  return (
+    <TableContainer component={Paper} className={classes.table}>
```

Comment:
You could consider using the Backstage `Table` component instead: http://storybook.backstage.io/?path=/story/table--default-table

Reply:
Hi @stefanalund sadly the link is broken. Do you know if it changed?

Reply:
Correct link: https://backstage.io/storybook/?path=/story/table--default-table